src/main/java/com/stuypulse/robot/MotorStall.java

```diff
@@ -0,0 +1,52 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018-2019 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package com.stuypulse.robot;
+
+import com.stuypulse.robot.subsystems.Funnel;
+
+import edu.wpi.first.wpilibj.Timer;
+
+public class MotorStall implements Runnable {
+        
+    private final Funnel m_Funnel;
+
+    private double start_encoder_value;
```

Comment:
These should all probably be taken in the constructor to some degree
You can also, in the interface, force the subsystem to have getter-methods for each constant